jenkins/ci.fedora.Dockerfile

```diff
@@ -30,7 +30,8 @@
 FROM quay.io/fedora/fedora:34
```

Comment:
Since we are making changes here, we should bump to the non-EOL Fedora 35.

Reply:
Your wish is my command.  (Be careful what you wish for....)